### PR TITLE
feat: polish toolbar accessibility

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: publish-frontend/src/styles/globals.css [hunk 1]
```diff
@@ -33,3 +33,6 @@
   margin: 0 0.5rem;
 }
 
+ul {
+  list-style-type: none;
+}
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply publish-frontend/src/styles/globals.css:38]: which do you prefer module or inline css?

[the remark this answers is not in the document]
Comment (Sembauke) [reply publish-frontend/src/styles/globals.css:38]: @Sboonny do what you think is best.

My thoughts is: 
1. If there are many elements that would need to be styled create a class. 
2. If there are only one or two classes, style it with an inline style.

File: publish-frontend/src/styles/globals.css [hunk 2]
```diff
@@ -32,4 +32,3 @@
   height: 2rem;
   margin: 0 0.5rem;
 }
```
Comment (Sboonny) [inline publish-frontend/src/styles/globals.css:34]:
```suggestion
}

```

File: publish-frontend/src/styles/globals.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sboonny) [reply publish-frontend/src/styles/globals.css]: make sense, I forgot about chakra props, which is similar to the second option, it should be solved now 👍.

**Review by Sembauke — APPROVED** [review]

**Comment by Sembauke** [comment]
Thanks Muhammed you're a legend

**Pull request merged**